spec/sidekiq/lighthouse/poll_form526_pdf_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Lighthouse::PollForm526Pdf, type: :job do
+  subject { described_class }
+
+  before do
+    Sidekiq::Job.clear_all
+    allow_any_instance_of(Auth::ClientCredentials::Service).to receive(:get_token).and_return('fake_access_token')
+  end
+
+  let(:user) { FactoryBot.create(:user, :loa3) }
+  let(:auth_headers) do
+    EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
+  end
+
+  describe '.perform_async' do
+    let(:saved_claim) { FactoryBot.create(:va526ez) }
+    let(:submitted_claim_id) { 600_130_094 }
```

Comment:
I don't see where `submitted_claim_id` is used in this test.

Reply:
copy/paste remnant. killed it.